tests/annotate/test_cell_calling.py

```diff
@@ -40,7 +41,7 @@ def test_find_component_min_size_limits_only_noise():
     test_data = random_state.poisson(100, 500)
 
     min_bound = find_component_size_limits(np.absolute(test_data), direction="lower")
-    assert min_bound < 100
+    assert min_bound >= MINIMUM_N_EDGES_CELL_SIZE
```

Comment:
Is this going to work now? Do we have `test_data` with cells over 300 edges. I ask because the sign before was the opposite actually.

Reply:
I revered it since now we will never find a minimum value smaller than the threshold, even when we only have noise - which is what this test tries to get to.